### DISCUSSION TITLE
Query an interface that Implements another interface

### DISCUSSION THREAD

**Discussion by superjose** [graphql-nexus/nexus#1121] [Q&A]
Sup everyone! 😉. First of all, extremely grateful what the Nexus team has accomplished so far. We've been using it in a production environment and it's been a blast. Code First and Type Safety for GraphQL is awesome. 

We haven't run into blocking issues, except for this one. I think it's better for us to open a discussion vs contaminating your PR. 

I posted [this question ](https://stackoverflow.com/questions/73127014/query-an-interface-that-implements-another-interface-in-graphql)on Stack Overflow.

Basically, we're trying to use multiple interfaces and call them in a single GraphQL query (please [see the Stack Overflow post for a detailed explanation](https://stackoverflow.com/questions/73127014/query-an-interface-that-implements-another-interface-in-graphql)). 

But when we try to use it in our GraphQL implementation it doesn't seem to work. 

To see exactly what we're trying to do, please visit this Code Sandbox:
* https://codesandbox.io/s/silly-kirch-o7bk2g
And run the following query:

```graphql
query getFeed {
    activityFeed {
        id
        name
        description
        ... on IMedia {
            type
            format
        }
        ... on INews {
            date
            author
            content
        }
        ... on Video {
            length
        }
    }
}
```

Trying that using Nexus doesn't work. We would receive empty fields. 

Here's our Nexus initialization (Note, that I'm omitting the import statements and most types that we have defined)

```ts
function generateConfig(): boolean | any {
  if (process.env.PLATFORM === 'graphql') {
    // eslint-disable-next-line @typescript-eslint/no-var-requires
    const path = require('path');
    return {
      contextType: {
        module: path.join(__dirname, '../../src/shared/common/context.ts'),
        export: 'GraphQLContext',
      },
      outputs: {
        schema: path.join(
          __dirname,
          '../../schema.generated.graphql'
        ) as string,
        typegen: path.join(
          __dirname,
          '../../node_modules/@types/nexus-typegen/index.d.ts'
        ),
      },
    };
  }

  return {
    shouldGenerateArtifacts: false,
    contextType: false,
    outputs: {
      schema: false,
      typegen: false,
    },
  };
}


export const schema = makeSchema({
  ...generateConfig(),
  plugins: [connectionPlugin()],
  features: {
    abstractTypeStrategies: {
      isTypeOf: true,
    },
  },
  types: [
  // Omitted
  ],
});
```

Is there a configuration that we're missing?

**Comment by Amaan215422** [comment]
Thanks for this, great work
